web-console/project-details/storage-metrics.md

```diff
@@ -0,0 +1,61 @@
+# Storage usage Metrics
+
+## Open Questions
+- What should the entry for capacity consumed be called? I'm using "Used" in these mocks, but should it be "Capacity Consumed"? Something else?
+- What would we want the thresholds for the donut chart to be? Are the defaults ok?
+- Will these storage metrics be tied to any events or warnings so that we can proactively inform users of any storage problems?
+- Is `NA` how we want to show that the metric doesn't exist? Should we add a tooltip to that as well?
+- Is the order of the fields ok for the PVC details page?
+
+## Storage List Page
+
+![storage list](img/storage-list.png)
+
+- Add a column **Used** between the **Status** and **Capacity** columns.
+- The units should match between the **Used** and **Capacity** columns for easy comparison.
+- Because it's possible that not all storage classes will have the capacity consumed metric, field-level help should be used in the column header to explain why some rows may not have an entry in that column.
+- If that metric is not available for a storage class, `NA` should appear in the **Used** column for that row.
+
+
+## Storage Details Page
+![storage details](img/storage-details.png)
+- When the capacity consumed metric is available for a PVC, a utilization donut chart is added to the details page to show the capacity consumed.
+- A **Used** entry should appear in the details list and the units should match the units for **Capacity**.
+- If the PVC has a storage class, that should be shown on the details page regardless whether metrics are available or not.
+
+
+#### Storage Donut Chart
+- Use [PatternFly Donut Chart](http://www.patternfly.org/pattern-library/data-visualization/donut-chart/#design).
+- The chart should be color coded according to thresholds. The default thresholds are:
+  - Green (`#3f9c35`) if < 75%
+  - Gold (`#ec7a08`) between 75-89%
+  - Red (`#cc0000`) >= 90%
+- The label in the center of the donut should give the capacity used in the fomat "x of y GiB", with the x value in large type.
```

Comment:
Just curious about Volume Details.  How will this look if the volume here has multiple PVCs?

Reply:
The donut represents the capacity consumed of a single PVC, on the details page for that PVC. 

If you're asking from the perspective of a deployment, where multiple PVCs are mounted as single a volume, I was not aware that that was possible. I would need some explanation to better understand that scenario.

Reply:
@ncameronbritt You're right, currently PVCs have a 1:1 binding to PVs.  My bad.  I got confused with something else.

+1 on donut representing single PVC consumption.  I was trying to ask I guess (in a very indirect way) whether we would not consider showing both used and available at the same time via the donut.